plugins/formal-verification/testData/diagnostics/bad_contracts/calls_in_place.fir.diag.txt

```diff
@@ -27,11 +27,10 @@ method global$fun_unknown$fun_take$fun_take$T_Int$return$T_Int$return$T_Int(loca
   ensures true
 
 
-/calls_in_place.kt:(450,472): warning: Viper verification error: Postcondition of global$fun_incorrect_at_most_once$fun_take$fun_take$T_Int$return$T_Int$return$T_Int might not hold. Assertion local$f.special$function_object_call_counter <= old(local$f.special$function_object_call_counter) + 1 might not hold. (<no position>)
-
-
 /calls_in_place.kt:(450,472): warning: Function incorrect_at_most_once may not satisfy its contract.
 
+/calls_in_place.kt:(521,550): warning: Viper verification error: Postcondition of global$fun_incorrect_at_most_once$fun_take$fun_take$T_Int$return$T_Int$return$T_Int might not hold. Assertion local$f.special$function_object_call_counter <= old(local$f.special$function_object_call_counter) + 1 might not hold. (<wrapped value>)
```

Comment:
Why do we want to know that there is a wrapped value position here?

Reply:
I use it for debugging purposes, so I can see immediately from the diagnostics if a position is missing

Reply:
In this case I would just make a new diagnostic that reports that a position is expected but missing.  It's more likely people will notice it and fix it when they make changes.